docs/group-by.rst

```diff
@@ -50,10 +54,15 @@ Groups the result by the given tags that are defined through the ``groupByKey``
 
     :param aggregation: aggregation type, one of ``area``, ``count``, ``length``, ``perimeter``
     :query <params>: see query-parameters_ at /elements(aggregation) endpoint
-    :query groupByKey: OSM key(s) given as a list and combined with the ‘AND’ operator, e.g.: 'highway’, 'building’; (one groupByKey parameter must be defined)
+    :query groupByKey: OSM key e.g.: 'highway’, 'building’; no default value (one groupByKey parameter must be defined)
```

Comment:
isn't the _one_ in _“(one groupByKey parameter must be defined)”_ superfluous and/or confusing, since there can and must be exactly one. better just say something like: _“(this parameter must be defined)”_?

Reply:
hmm.. yeah this could be rephrazed a bit.. your suggestion sounds fine. Let's see then what @SlowMo24 says :)

Reply:
I added my version in commit. I like to be more verbose. giving an example with two keys while only one can be specified seemed confusing for me

Reply:
one might also think about the default: no value. although it is technically correct semantically it means 'all values' ;-)